docs/source/algorithms/CropWorkspaceRagged-v1.rst

```diff
@@ -9,19 +9,44 @@
 Description
 -----------
 
-This is a workflow algorithm that crops each spectrum of a workspace
-independently. This is intended for workspaces with a relatively small
-number of spectra (e.g. <10), but places no restrictions on the input
-workspace.
+This is an algorithm that crops each spectrum of a workspace
+independently.
 
 The minimum and maximum values that are specified are interpreted as follows:
 
 * One value per spectrum. If there is only one value overall, it is used for all of the spectra.
-* ``numpy.nan``, ``math.nan``, and ``np.inf`` are interpreted to mean use the data's minimum or maximum x-value.
+* If the given value is smaller than the first data point, it will use the first data point.
+* If the given value is larger than the last data point, it will use the last data point.
 
 Usage
 -----
 
+**Example - Crop a workspace.**
+
+.. testcode:: CropRagged
+
+    ws = CreateWorkspace([0,1,2,3,4,2, 3,4,5,6],[0,1,2,3,4,3,4,5,6,7], NSpec=2)
+    x_min=[]
+    x_max =[]
+
+    for j in range(ws.getNumberHistograms()):
+        x_min.append(1.0+j)
+        x_max.append(14.0)
+    
+    new=CropWorkspaceRagged(ws,x_min,x_max)
+
+    print("The number of bins in spectrum 1 is: {}".format(new.readX(0).size))
+    print("The number of bins in spectrum 2 is: {}".format(new.readX(1).size))
+
+Output:
+
+.. testoutput:: CropRagged
+
+    The number of bins in spectrum 1 is: 4
+    The number of bins in spectrum 2 is: 5
+
+**Example - Crop a workspace as part of a workflow.**
+
 .. include:: ../usagedata-note.txt
 
 This is an example of how ``CropWorkspaceRagged`` would be used near
```

Comment:
numpy.nan and math.nan are no longer mentioned above. Also the second example no longer runs and crashes mantid

Reply:
huh, it was ok when I tried that. Strange, it did take me ages to get the data

Reply:
CropWorkspaceRagged works fine, it falls over for me when doing the PDFFourierTransform algorithm

Reply:
ah sorry, I was thinking of a different script. Yes this falls over for me too. It looks like that algorithm expects the array to be longer (probably padded with zeros. like it use to be). It is a script from the SNS so I have asked them for advice.